website/data/docs-nav-data.json

```
@@ -544,6 +564,10 @@
         "title": "<code>Request Limiter</code>",
         "path": "configuration/request-limiter"
       },
+      {
+        "title": "<code>Adaptive Overload Protection</code>",
```

```suggestion
        "title": "Adaptive overload protection",
```

Code formatting should be reserved for actual config parameters and other code. The title, as written, is not code